### ISSUE TITLE
Can it also handle/process p2sh addresses starting with '3' and bech32 segwit addresses starting with 'b'

### ISSUE THREAD

**Issue by alfredalappat** [FiberMan/bruteforceBTC#2]
I have a doubt that is this code only accept legacy address or also can handle p2sh addresses and segwit addresses starting with 'b'?. please respond.

**Comment by FiberMan** [comment]
I don't think it will work with other types. But you can check it by providing a known private key in the [script](https://github.com/FiberMan/bruteforceBTC/blob/676081800c0fec57a5fb4c6d010b3e8d4b920671/brute.py#L46) and appropriate address in `addresses.txt`.
